src/KDTrack.C

```diff
@@ -191,20 +195,17 @@ double KDTrack::calculateChi2SZ(TH2F* histo, bool debug) {
     double ds2    = (errorS2 + m_gradientZS * m_gradientZS * errorZ * errorZ);
 
     // Increment the chi2
-    chi2 += (residualS * residualS) / (ds2);
+    chi2 += (residualS * residualS) / ds2;
 
     // Debug info
-    if (debug)
+    if (debug) {
       streamlog_out(DEBUG5) << "- hit " << hit << " has residualS = " << residualS << ", with error dz = " << errorZ
                             << ", error ds = " << sqrt(errorS2) << " and total error = " << sqrt(ds2) << std::endl;
-
-    double debugError2 = sqrt(deltaPhi * deltaPhi * errorRadC * errorRadC + errorDeltaPhi * errorDeltaPhi * radC * radC);
-    if (debug)
+      double debugError2 = sqrt(deltaPhi * deltaPhi * errorRadC * errorRadC + errorDeltaPhiSquared * radC * radC);
```

Comment:
minor: since it is a sqrt, why is this named Error2?

Reply:
the name was there before, but I will rename it